src/particles.cpp

```diff
@@ -166,16 +183,32 @@ void Particle::updateLight()
 	else
 		light = blend_light(m_env->getDayNightRatio(), LIGHT_SUN, 0);
 
-	m_light = decode_light(light);
+	m_light = decode_light(light + m_glow);
 }
 
 void Particle::updateVertices()
 {
 	video::SColor c(255, m_light, m_light, m_light);
-	f32 tx0 = m_texpos.X;
-	f32 tx1 = m_texpos.X + m_texsize.X;
-	f32 ty0 = m_texpos.Y;
-	f32 ty1 = m_texpos.Y + m_texsize.Y;
```

Comment:
There is a multiply with m_texsize (`framesize_f.X * m_texsize.X`) so this isn't really useful IMO